tests/selenium/single-page-stripe-test.js

```diff
@@ -148,10 +145,9 @@ module.exports = function(driver, By, done) {
     });
   });
 
-  driver.get('http://localhost:3000/pa-IN/?test=frequency-move-test&billingAddress=partial');
+  driver.get('http://localhost:3000/pa-IN/partial-page/?test=frequency-move-test');
```

Comment:
yeah, running this on travis would have caught it, but still not ready for that.